gobblin-modules/gobblin-crypto-provider/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
 
   testCompile project(":gobblin-test-utils")
   testCompile externalDependency.testng
+  testCompile project(path: ":gobblin-modules:gobblin-crypto", configuration: "tests")
```

Comment:
This is to pull in stuff like GPGFileEncryptorTest.PUBLIC_KEY?

Reply:
Yes, this is to get the shared test config.